javascript/features/nuwani_commands/communication_commands.js

```diff
@@ -113,6 +118,15 @@ export class CommunicationCommands {
         context.respond('5LVP Discord: https://discord.sa-mp.nl/');
     }
 
+    // !help
+    //
+    // Displays information on how people can join our Discord channel.
+    onHelpCommand(context) {
+        context.respond('5Available IRC commands: !getid, !getname, !msg, !players, !pm, !vip, !discord');
+        context.respond('To register an account, visit: www.sa-mp.nl');
```

Comment:
Thanks! Could we lower this to two messages?

I think the one on line 125 is fine. We can combine lines 126 and 127 by using:

```javascript
context.respond('Register for an account on https://sa-mp.nl/, and use the in-game "/account" command to change your name, password or settings.');
```